otter/test/integration/test_lib.py

```diff
@@ -0,0 +1,137 @@
+"""
+Tests for the lib functions for convergence black-box testing.
+"""
+
+from twisted.internet.defer import Deferred
+from twisted.trial.unittest import SynchronousTestCase
+
+from otter.test.integration import lib
+
+
+class IdentityV2Tests(SynchronousTestCase):
+    """Tests for the :class:`IdentityV2` class."""
+
+    def test_missing_params(self):
```

Comment:
I include this test to provide concrete documentation of expected behavior.  You're right that, semantically, it's basically testing characteristic.  However, unit tests double as documentation (executable specifications), and this test falls under that particular use-case.